web/web.go

```diff
@@ -0,0 +1,771 @@
+		l = append(l, stat{s, x.Schema + x.Data + x.Skip + x.Error})
+	}
+	// Sort by alphabetical order of statements.
+	sort.Slice(l, func(i, j int) bool {
+		return l[i].statement < l[j].statement
+	})
+	stmtstats = stmtstats + writeHeading("Statements Processed")
+	stmtstats = stmtstats + "Analysis of statements in " + driverName + " output, broken down by statement type.\n"
+	stmtstats = stmtstats + "  schema: statements successfully processed for Spanner schema information.\n"
+	stmtstats = stmtstats + "    data: statements successfully processed for data.\n"
+	stmtstats = stmtstats + "    skip: statements not relevant for Spanner schema or data.\n"
+	stmtstats = stmtstats + "   error: statements that could not be processed.\n"
+	stmtstats = stmtstats + "  --------------------------------------\n"
+	stmtstats = stmtstats + fmt.Sprintf("  %6s %6s %6s %6s  %s\n", "schema", "data", "skip", "error", "statement")
+	stmtstats = stmtstats + "  --------------------------------------\n"
+	for _, x := range l {
+		s := conv.Stats.Statement[x.statement]
+		stmtstats = stmtstats + fmt.Sprintf("  %6d %6d %6d %6d  %s\n", s.Schema, s.Data, s.Skip, s.Error, x.statement)
+	}
+	if driverName == "pg_dump" {
+		stmtstats = stmtstats + "See github.com/lfittl/pg_query_go/nodes for definitions of statement types\n"
+		stmtstats = stmtstats + "(lfittl/pg_query_go is the library we use for parsing pg_dump output).\n"
+		stmtstats = stmtstats + "\n"
+	} else if driverName == "mysqldump" {
+		stmtstats = stmtstats + "See https://github.com/pingcap/parser for definitions of statement types\n"
+		stmtstats = stmtstats + "(pingcap/parser is the library we use for parsing mysqldump output).\n"
+		stmtstats = stmtstats + "\n"
+	}
+	return stmtstats
+}
+func getOverview(w http.ResponseWriter, r *http.Request) {
+	reports := internal.AnalyzeTables(app.conv, nil)
+	summary := internal.GenerateSummary(app.conv, reports, nil)
+	overview := writeHeading("Summary of Conversion")
+	overview = overview + summary + "\n"
+	ignored := internal.IgnoredStatements(app.conv)
+	if len(ignored) > 0 {
+		overview = overview + fmt.Sprintf("Note that the following source DB statements "+
+			"were detected but ignored: %s.\n\n",
+			strings.Join(ignored, ", "))
+	}
+	statementsMsg := ""
+	var isDump bool
+	if strings.Contains(app.driver, "dump") {
+		isDump = true
+	}
+	if isDump {
+		statementsMsg = "stats on the " + app.driver + " statements processed, followed by "
+	}
+	overview = overview + "The remainder of this report provides " + statementsMsg +
+		"a table-by-table listing of schema and data conversion details. " +
+		"For background on the schema and data conversion process used, " +
+		"and explanations of the terms and notes used in this " +
+		"report, see HarbourBridge's README.\n\n"
+	if isDump {
+		overview = overview + writeStmtStats(app.driver, app.conv)
+	}
+	w.WriteHeader(http.StatusOK)
+	json.NewEncoder(w).Encode(overview)
+}
+
+type severity int
```

Comment:
These definitions (severity and the consts warning and note) don't seem to be used within package web?
Can we just delete them?

If they are used, since they are identical to ones in internal -- shouldn't we just make the ones in internal public and use those?

Reply:
they are not needed, removed